WasteManagement/README.md

```diff
@@ -0,0 +1,4 @@
+# Grasse Use Case NGSI-Ld Data Model
```

Comment:
it would be easier for readers to put the contents of the GrasseUseDataModel.docx into this README